docs/auth/auth-backend-classes.md

```diff
@@ -61,6 +61,19 @@ If your authentication provider is any of the above mentioned providers, you can
 configure them by setting the right variables in `app-config.yaml` under the
 `auth` section.
 
+### SAML
+
+The SAML Provider is currently under development. Additional validation and
+profile handling is still required before use in production.
+
+To configure the SAML Auth provider, look at the configuration parameters
```

Comment:
It's not properly filled in for many providers yet, but a good way to document the options is to add them to the config schema in `plugins/auth-backend/config.d.ts`

Reply:
That's a great idea, let me do that.